lib/actions/apiV2.js

```diff
@@ -1209,7 +1209,8 @@ export function routingQuery(searchId = null, updateSearchInReducer) {
               ) {
                 // Add accessibility error, if it is turned on
                 const state = getState()
-                const { displayA11yError } = state.otp.config?.itinerary
+                const { displayA11yError, displayMicromobilityRentalError } =
+                  state.otp.config?.itinerary
```

Comment:
I see why you called this error, since we use the error renderer to show it, but I think it would be better to call it a warning. From the perspective of someone writing a config file for OTP-RR who doesn't know the implementation details this feels more like a warning than an error, right?

Reply:
Hmm I changed this in [ce63aa0](https://github.com/opentripplanner/otp-react-redux/pull/1413/commits/ce63aa0e2197f11b3e06ce041b1e328dc67e014e), but curious if the 2nd reviewer has an opinion. I feel like it might be useful to keep the naming conventions for all the "errors" the same, so that it's clear this is being displayed by the error renderer, rather than some other component?

Reply:
If we renamed the error renderer to Alert or something maybe that would be better. I just don't think config items should be dictated by the name of the component that renders them, that's an implementation detail that isn't important to the end user.

Reply:
Yeah can we make this consistent with the `displayA11yError`? Both in terms of labeling, implementation, and even the comment in the example-config.yml